src/useKeyboardEvent/useKeyboardEvent.stories.mdx

```diff
@@ -0,0 +1,74 @@
+import { Meta } from '@storybook/addon-docs';
+
+<Meta
+  title="useKeyboardEvent/docs"
+/>
+
+# useKeyboardEvent
+
+The `useKeyboardEvent` hook uses the [useEventListener](?path=/story/useeventlistener-docs--page) to attach a keyboard event (`keydown`) to a specific
+target and triggers a callback when the provided key is pressed.
+
+The default target is the `document`, because local bindings are usually done through the `event`
+binding.
+
+But if you ever need to you could still overwrite the event target to something else, it
+has support for the following targets:
+* `Window`
+* `Document`
+* `DomElementOrRef` - Meaning all types of Dom elements or Muban Refs
+
+> **Note:** You can use the [useEscapeKeyEvent](?path=/story/useescapekeyevent-docs--page) hook if you want to attach a listener for the `Escape` key
+
+## Reference
+
+```ts
+function useKeyboardEvent(
+  keys: string | Array<string>,
+  callback: KeyDownCallback,
+  target: EventTarget = document,
+): void
+```
+
+### Parameters
+
+* `keys` – The key(s) that you want to listen for.
+* `callback` – The function to invoke when the one of the keys is pressed.
+* `target` – The target on which the listener is bound, by default the document.
+
+> **Note:** We encourage using something like [ts-key-enum](https://www.npmjs.com/package/ts-key-enum)
+> for the `keys` as much as possible, this way there is less room for typo's.
+
+### Returns
+
+* `void`
+
+## Usage
+
+```ts
+useKeyboardEvent(Key.ArrowRight, (event: KeyboardEvent) => {
```

Comment:
Just wondering if:
* We should also have an example of using this without the `ts-key-enum`?
* And/or include the `import Key from "ts-key-enum"` in this usage example – and the snippet below as well?

Reply:
Updated the docs to be more descriptive about the usage of the `ts-key-enum`!